test/messagesIncoming.ts

```diff
@@ -848,30 +908,48 @@ describe('Incoming Messages', async () => {
 			prevBlockNodes = constructTree(prevBlockIds);
 		});
 
-		it('Should not be able to pause as non-owner', async () => {
+		it('Should be able to grant pauser role', async () => {
+			expect(await env.fuelMessagePortal.hasRole(pauserRole, env.addresses[2])).to.equal(false);
+
+			// Grant pauser role
+			await expect(env.fuelMessagePortal.grantRole(pauserRole, env.addresses[2])).to.not.be.reverted;
+			expect(await env.fuelMessagePortal.hasRole(pauserRole, env.addresses[2])).to.equal(true);
+		});
+
+		it('Should not be able to pause as non-pauser', async () => {
 			expect(await env.fuelMessagePortal.paused()).to.be.equal(false);
 
 			// Attempt pause
 			await expect(env.fuelMessagePortal.connect(env.signers[1]).pause()).to.be.revertedWith(
-				'Ownable: caller is not the owner'
+				`AccessControl: account ${env.addresses[1].toLowerCase()} is missing role ${pauserRole}`
 			);
 			expect(await env.fuelMessagePortal.paused()).to.be.equal(false);
 		});
 
-		it('Should be able to pause as owner', async () => {
+		it('Should be able to pause as pauser', async () => {
 			expect(await env.fuelMessagePortal.paused()).to.be.equal(false);
 
 			// Pause
-			await expect(env.fuelMessagePortal.pause()).to.not.be.reverted;
+			await expect(env.fuelMessagePortal.connect(env.signers[2]).pause()).to.not.be.reverted;
+			expect(await env.fuelMessagePortal.paused()).to.be.equal(true);
+		});
+
+		it('Should not be able to unpause as only pauser', async () => {
```

Comment:
I read this as if it meant that under the condition that there is only 1 pauser then we cannot unpause rather than if they do not have the admin role

Reply:
Reworded to `Should not be able to unpause as pauser (and not admin)`